CRM/Utils/String.php

```diff
@@ -638,10 +638,14 @@ public static function purifyHTML($string) {
       $config = HTMLPurifier_Config::createDefault();
       $config->set('Core.Encoding', 'UTF-8');
       $config->set('Attr.AllowedFrameTargets', ['_blank', '_self', '_parent', '_top']);
-
       // Disable the cache entirely
       $config->set('Cache.DefinitionImpl', NULL);
-
+      $config->set('HTML.DefinitionID', 'enduser-customize.html tutorial');
+      $config->set('HTML.DefinitionRev', 1);
+      if ($def = $config->maybeGetRawHTMLDefinition()) {
```

Comment:
I find it really hard to tell if it's a typo or deliberate when I see this construct - my IDE would insist on 
``` if (($def = $config->maybeGetRawHTMLDefinition()) !== FALSE) {```
- although I would prob split up the assign & if